doc/source/tracking/paper.md

```diff
@@ -0,0 +1,159 @@
+aas-journal:
+---
+
+# Summary
+
+icepyx is both a software library and a community composed of ICESat-2 (NASA satellite) data users, developers, maintainers, and the scientific community.
+We are working together to develop a shared library of resources - including existing resources, new code, tutorials, and use-cases/examples - that simplify the process of querying, obtaining, analyzing, and manipulating ICESat-2 datasets to enable scientific discovery.
+
+# Statement of need
+
+icepyx aims to provide a clearinghouse for code, functionality to improve interoperability, documentation, examples, and educational resources that tackle disciplinary research questions while minimizing the amount of repeated effort across groups utilizing similar datasets.
+icepyx also hopes to foster collaboration, open-science practices, and reproducible workflows by integrating and sharing resources.
+
+The Ice, Cloud, and Land Elevation Satellite-2 (ICESat-2) [@is2] was launched by NASA in September 2018.
+The laser altimeter on board the satellite emits green light to the Earth's surface and measures the time until each pulse is returned to the satellite's sensors.
+This information is used to determine the surface height of the land, ice, snow, trees, water, clouds, etc. that the satellite is passing over.
+The instrument provides close to 500 GB of data per day, allowing scientists to investigate the surface height of earth's features in unprecedented detail.
+
+icepyx began during the cryosphere-themed ICESat-2 Hackweek at the University of Washington in June 2019.
+At the event, there was a clear need for a collaborative, shared community space that combined and generalized the tools and materials written by past, present, and future Hackweek participants, ICESat-2 Science Team members, and the data user community.
+A unified framework of code and documentated examples for downloading, reading, and visualizing ICESat-2 data that is well tested makes it more accessible for everyone to use.
+The library and community continue to grow and evolve, adding new features and building scientific literacy in open-science, cloud computing, and collaborative development best practices.
+
+icepyx is now a foundational tool for accessing and working with ICESat-2 data, responsible for nearly a quarter of all NASA data center granule downloads in 2022. The library is complimented by a series of other specialized tools for interacting with and obtaining ICESat-2 data. These include OpenAltimetry[@OA], a browser based web tool to visualize and download selected ICESat and ICESat-2 surface heights, SlideRule[@SR], a server-side framework to create cloud-based, on-demand customized data processing for ICESat-2 data, and multiple product-focused tools for scientific data analysis (e.g. PhoREAL, PhotonLabeler). These tools are described in more detail in the icepyx documentation's [ICESat-2 Resource Guide](https://icepyx.readthedocs.io/en/latest/community/resources.html).
+
```

Comment:
@weiji14 Are you available to look over this updated text before I note the updates in the JOSS review?

Reply:
One typo spotted. Just off the top of my head, I would maybe include Tyler's https://github.com/tsutterley/read-ICESat-2? I found a link to PhoREAL at https://github.com/icesat-2UT/PhoREAL but couldn't find one for PhotonLabeler, is it https://github.com/Oht0nger/PhoLabeler?

```suggestion
icepyx is now a foundational tool for accessing and working with ICESat-2 data, responsible for nearly a quarter of all NASA data center granule downloads in 2022. The library is complemented by a series of other specialized tools for interacting with and obtaining ICESat-2 data. These include OpenAltimetry[@OA], a browser based web tool to visualize and download selected ICESat and ICESat-2 surface heights, SlideRule[@SR], a server-side framework to create cloud-based, on-demand customized data processing for ICESat-2 data, and multiple product-focused tools for scientific data analysis (e.g. PhoREAL, PhotonLabeler). These tools are described in more detail in the icepyx documentation's [ICESat-2 Resource Guide](https://icepyx.readthedocs.io/en/latest/community/resources.html).

```
On the last sentence, I feel like there was a good NASA Earthdata or NSIDC page that lists out all the ICESat-2 data tools, but can't seem to find it anymore.

Reply:
We may not need to include this but the Bathy working group has a GitHub site with tools too - https://github.com/ICESat2-Bathymetry/Information

Jessica, I found a separate tools link from the one you sent - https://nsidc.org/data/user-resources/help-center/icesat-2-tools-and-services

Reply:
> We may not need to include this but the Bathy working group has a GitHub site with tools too - https://github.com/ICESat2-Bathymetry/Information
> 
> Jessica, I found a separate tools link from the one you sent - https://nsidc.org/data/user-resources/help-center/icesat-2-tools-and-services

Thanks - these are great resources, Kelsey! I am adding them to #412.

Reply:
Yes, https://nsidc.org/data/icesat-2/tools was the one I was looking for. Thanks for updating the draft pdf and opening https://github.com/icesat2py/icepyx/issues/412!